src/core_plugins/elasticsearch/lib/determine_supported_features.js

```diff
@@ -0,0 +1,25 @@
+import { checkFieldStatsApi } from './feature_checks';
+
+
+const featureChecks = [
+  checkFieldStatsApi,
+];
+
+export async function determineSupportedFeatures(callAdminAsKibanaUser) {
+  const { nodes: nodesInfo } = await callAdminAsKibanaUser('nodes.info', {
```

Comment:
anything not used only internally (health checks, etc) should be callWithRequest imo.  i think it makes sense to proxy here